jobs/jailbreak_paris_dbnomics_python.html

```diff
@@ -0,0 +1,45 @@
+---
+title: DevOps, révolutionnez l'économie par le logiciel libre et l'open data !
```

Comment:
We prefer 'title' to be just the name of the role.

Reply:
Thanks, I corrected the error.